support-frontend/app/services/HttpIdentityService.scala

```diff
@@ -58,12 +59,23 @@ object IdentityService {
 }
 
 case class GetUserTypeResponse(userType: String)
-
 object GetUserTypeResponse {
   implicit val readsGetUserTypeResponse: Reads[GetUserTypeResponse] = Json.reads[GetUserTypeResponse]
   implicit val getUserTypeEncoder: Encoder[GetUserTypeResponse] = deriveEncoder
 }
 
+case class CreateSignInTokenResponse(encryptedEmail: String)
+object CreateSignInTokenResponse {
+  implicit val readCreateSignInTokenResponse: Reads[CreateSignInTokenResponse] = Json.reads[CreateSignInTokenResponse]
+  implicit val createSignInTokenResponseEncoder: Encoder[CreateSignInTokenResponse] = deriveEncoder
+}
+
+case class CreateSignInLinkResponse(signInLink: String)
```

Comment:
It looks like this belongs in `IdentityController`, since that's where it's used (as the response type).
Also, you don't need `readCreateSignInLinkResponse` in this case, because we use circe to serialize the response

Reply:
Done ✅ thanks